discoverx/table_info.py

```diff
@@ -88,16 +161,15 @@ def _get_table_list_sql(self, catalogs: str, schemas: str, tables: str, columns:
             match_any_col = "|".join([f'({c.replace("*", ".*")})' for c in columns])
             columns_sql = f"""AND regexp_like(column_name, "^{match_any_col}$")"""
 
-        sql = f"""
+        with_column_info_sql = f"""
```

Comment:
Wouldn't it make sense to retrieve the table list using python instead of SQL in the future? Might be much cleaner compared to the current solution. And I guess we don't need any `.explain()` method for retrieving table-infos anyway.